apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createservice user.RegisterNewServiceUserService`
+ */
+const faker = require('faker')
+const { makeLoggedInAdminClient } = require('@core/keystone/test.utils')
+const {
+    registerNewServiceUserByTestClient,
+    createTestUser,
+    createTestEmail,
+    makeLoggedInClient,
+    makeClientWithSupportUser,
+} = require('@condo/domains/user/utils/testSchema')
+const { EMAIL_ALREADY_REGISTERED_ERROR } = require('@condo/domains/user/constants/errors')
+const { REGISTER_NEW_SERVICE_USER_MUTATION } = require('@condo/domains/user/gql')
+
+describe('RegisterNewServiceUserService', () => {
+    test('register new service user', async () => {
+        const client = await makeLoggedInAdminClient()
+        const name = faker.fake('{{name.suffix}} {{name.firstName}} {{name.lastName}}')
```

Comment:
I think, if this one is repeated a lot of times its better to move it away to utils. 

BTW, doesnt condo have something like this? Generating random user name seems like a generic task